docs/xchain.js

```diff
@@ -1033,3 +1040,36 @@ function makeRawRequestInit(stringBody) {
     body: stringBody,
   };
 }
+
+function fillNetworkIdFromQueryString() {
+  const requestKey = getParameterByName('network');
+  if (requestKey) {
+    document.getElementById('pact-id').value = requestKey;
+  }
+}
+
+function getParameterByName(name, url = window.location.href) {
+  name = name.replace(/[\[\]]/g, '\\$&');
+  const regex = new RegExp(`[?&]${name}(=([^&#]*)|&|#|$)`);
+  const results = regex.exec(url);
+  if (!results) return null;
```

Comment:
Should we also return an empty string here?

Reply:
Copilot told me so :')

Reply:
Alrighty 👌